frontend/src/store/store.ts

```diff
@@ -74,6 +91,11 @@ export function configureStore(preloadedState?: any) {
     ),
   )
 
+  // automatically sync store.session.token with localstorage
+  store.subscribe(() => {
+    setAuthState(store.getState())
+  });

```

Comment:
hmmmmmm, I'm wondering if we can `useSelector` here to avoid passing the entire stateTree to `setAuthState`. I'm pretty sure selectors need to be within a react component (or at least, the child of a `Provider` instance). Would you mind trying to use a selector here & seeing if it explodes?

Reply:
Can't use `useSelector` outside of a react component, I tried that first.  I can research to see if there is a better way to do this.

Reply:
nah, it's fine. Let's just keep an eye on compiled app performance. If it degrades we'll come back here & see if it's causing problems